### PR TITLE
[BETA] Fixed fullview in watch data

### PR DESCRIPTION
**Pull request by jenow**
Fixed a bug in `watch data` where we couldn't have access to a fullview of a persisted document by clicking on the eye icon

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -31,11 +31,11 @@
...
-    "leaflet": "0.7.7",
-    "kuzzle-sdk": "1.7.3",
+    "leaflet": "^0.7.7",
     "angular-scroll-glue": "2.0.6",
-    "angular-cookies": "1.5.1",
+    "angular-cookies": "1.5.5",
     "highcharts": "4.2.3",
-    "oclazyload": "1.0.9"
+    "oclazyload": "1.0.9",
+    "kuzzle-sdk": "1.7.1"
```
Comment (scottinet) [inline bower.json]: Why has the SDK been downgraded to 1.7.1?

Comment (xbill82) [inline bower.json]: Plus, the SDK should be now installed via Npm, no?

**Comment by xbill82** [comment]
otherwise :+1:

**Comment by dbengsch** [comment]
+1

**Comment by stafyniaksacha** [comment]
+1 & merge

**Pull request merged**